module-apps/apps-common/widgets/TimeFixedWidget.hpp

```diff
@@ -21,21 +48,26 @@ namespace gui
                         const uint32_t &h,
                         const bool minus = false);
 
-        void setFirst(uint32_t first);
-        void setSecond(uint32_t second);
-        void setMinus(bool minus);
-        void setFont(const UTF8 &fontName);
+        void setMinutesBox(uint32_t first) const;
+        void setSecondsBox(uint32_t second) const;
```

Comment:
```suggestion
        void setMinutesBox(std::uint32_t first) const;
        void setSecondsBox(std::uint32_t second) const;
```
I believe we want to use C++ version, not the old C one.